src/Arcus.Scripting.Storage.Table/Scripts/Create-AzTableStorageAccountTable.ps1

```diff
@@ -47,6 +47,7 @@ if ($TableName -in $tables.Name) {
     if ($Recreate) {
         Write-Verbose "Deleting existing Azure storage table '$TableName' in the Azure storage account '$StorageAccountName'..."
         $isRemoved = Remove-AzStorageTable -Name $TableName -Context $storageAccount.Context -Force
+        Start-Sleep -Seconds 40
```

Comment:
Should we add this here? Or did you forgot to remove it? Best that we not fix or enforce this in our scripts, no?

Reply:
Don't really know where else to fix this.

The integration tests were consistently failing here, the docs state that a remove action can take 40 seconds: [learn.microsoft.com/en-us/rest/api/storageservices/Delete-Table?redirectedfrom=MSDN#remarks](https://learn.microsoft.com/en-us/rest/api/storageservices/Delete-Table?redirectedfrom=MSDN#remarks)

> Note that deleting a table is likely to take at least 40 seconds to complete. If you attempt an operation against the table while it's being deleted, the service returns status code 409 (Conflict).

So I added a 40 second wait time and the integration test now works.
Other option could be to maybe increase the default max wait timeout? https://github.com/pim-simons/arcus.scripting/blob/7fea36b34fa0f46334c589b40ba16fe6ce1f3f45/src/Arcus.Scripting.Storage.Table/Scripts/Create-AzTableStorageAccountTable.ps1#L7

Reply:
Ok, wow, 40s is then indeed something that could happen; but we should maybe make this more smart by polling during that time to see if it was created, instead on hard-coding this?

Reply:
We could increase the default `MaxRetryCount` (https://github.com/pim-simons/arcus.scripting/blob/7fea36b34fa0f46334c589b40ba16fe6ce1f3f45/src/Arcus.Scripting.Storage.Table/Scripts/Create-AzTableStorageAccountTable.ps1#L7) to `15` or `20`, that should (in combination with default of 5 seconds for the `RetryIntervalSeconds`) fix it as well.